### DISCUSSION TITLE
How to test changes to onboarding PR using the CLI

### DISCUSSION THREAD

**Discussion by michaelmerrild** [renovatebot/renovate#17030] [Request Help] [answered]
I have installed the Renovate Github App, and the onboarding PR has been created.

To verify changes made to the onboarding configuration, I would like to complete a dry run using the Renovate CLI. I'm aware that Renovate eventually updates the PR body describing the effects of the config, but I'm looking for a faster feedback loop.

I have tried the following command:

```
RENOVATE_BASE_BRANCHES="renovate/configure" renovate --dry-run=full --use-base-branch-config=merge my-org/my-repo
```

But get the following error:

```
ERROR: Error fetching config file from base branch - possible config name mismatch between branches? (repository=my-org/my-repo)
       "configFileName": undefined,
       "baseBranch": "renovate/configure"
```

This seems to be because renovate.json must exist in both the default branch and base branch.

Is there any way to complete a dry run using only the configuration in the onboarding branch?

**Comment by viceice** [comment]
no

**Comment by rarkins** [comment, marked as answer]
You might be able to achieve "something" this way:
- All your config must be in your local `config.js` file where you're running Renovate (it won't read from the onboarding branch)
- Set onboarding=false and requireConfig=ignored (so lack of `renovate.json` in the default branch is ignored)
- Run with dryRun=full

It's not exactly the same, but might achieve what you're after.

**Comment by tewfik-ghariani** [comment]
Alternatively, you may set the "renovate/configure" temporarily as a default branch and proceed with the testing ;)
